### ISSUE TITLE
Application switching problem

### ISSUE THREAD

**Issue by pawelsu8** [GrapheneOS/os-issue-tracker#1507]
The problem has been occurring for over a week. I am using gesture support. When I swipe up quickly from the bottom of the screen the app closes. When I move my finger more slowly from the bottom of the screen up, and then hold my finger down to bring up the apps to switch them, there was usually a brief vibration after a while and then I could swipe to move to the appropriate window.
This has been working sporadically for more than a week now, and it's often the case that when I make a longer movement with my finger and hold it down on the screen to switch to another app, it displays a screen with windows of open apps only until unfortunately I don't get the characteristic vibration, and after I lift my finger

**Comment by thestinger** [comment]
Not a GrapheneOS issue.

**Comment by pawelsu8** [comment]
I don't understand your answer.
The problem, began to occur after installing the Grapheneos update and occurs generally in GrapheneOs system. The problem is related to the operation of the navigation gesture support function. Why are you blurting out a ticket with the problem and not verifying the problem? If the problem is not related to Grapheneos then with what?

**Comment by thestinger** [comment]
We don't try to track every Android issue here only ones causes by our changes or which we consider special and high priority.
